src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,57 @@
-export const Tabs = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { Tab } from '../../type/Tab';
+
+interface Props {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected: CallableFunction
+}
+
+export const Tabs:React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const isSelectedTab = tabs
+    .find(tab => tab.id === selectedTabId) || tabs[0];
```

Comment:
It's a full tab, not a `boolean` value
```suggestion
  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
```